internal/liqonet/tunnel-operator/tunnel-operator.go

```diff
@@ -23,20 +23,20 @@ import (
 	"strings"
 	"time"
 
+	netv1alpha1 "github.com/liqotech/liqo/apis/net/v1alpha1"
```

Comment:
Lint should follow [goimports](https://pkg.go.dev/golang.org/x/tools/cmd/goimports).